gulpfile.js

```diff
@@ -184,14 +193,14 @@ function changelog(cb) {
 			/\[[\da-f]+(?:, *[\da-f]+)*\]/g,
 			m => m.replace(/([\da-f]{7})[\da-f]*/g, '[`$1`](https://github.com/PrismJS/prism/commit/$1)')
 		),
-		dest('.')
+		dest('.'),
 	], cb);
 }
 
 const components = minifyComponents;
 const plugins = series(languagePlugins, minifyPlugins);
 
-
+exports.lint = lint;
 exports.watch = watchComponentsAndPlugins;
-exports.default = parallel(components, plugins, componentsJsonToJs, build);
+exports.default = series(parallel(components, plugins, componentsJsonToJs, build), lint);
```

Comment:
I’ve made the&nbsp;`lint` task depend on&nbsp;everything in&nbsp;`default`, since if&nbsp;the&nbsp;`prism.js` file hasn’t finished building yet and the&nbsp;`lint` task tries to&nbsp;lint&nbsp;it while it’s&nbsp;being&nbsp;built, it will cause the&nbsp;build to&nbsp;fail.

Reply:
We probably shouldn't lint `prism.js`, as that's a generated file. That said we should probably not build if `lint` errors, right?